llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1627,6 +1647,12 @@ void AddressSanitizer::instrumentMaskedLoadOrStore(
 
   IRBuilder IB(I);
   Instruction *LoopInsertBefore = I;
+
+  if(CLAsanDormant){
+    LoopInsertBefore = SplitBlockAndInsertIfThen(IB.CreateNot(IB.CreateLoad(IB.getInt1Ty(), DormantAsanFlag)), LoopInsertBefore, false);
```

Comment:
Why is this called "LoopInsertBefore" ? I don't see references to loops around this code.

Reply:
oh yes ignore me I thought you introduced it that variable.